packages/devtools_app/test/test_infra/scenes/memory/default.dart

```diff
@@ -182,5 +182,10 @@ class MemoryDefaultScene extends Scene {
     await tester.pumpAndSettle();
   }
 
+  Future<void> goToTraceTab(WidgetTester tester) async {
```

Comment:
where is this used in this PR?

Reply:
It is not used. Will use it in next PR for testing.